Azordev/did-admin-panel
# feat: a list of products is available

### Description

I have added a list of providers products

fixes #105 

#### Screenshots
![image](https://user-images.githubusercontent.com/82967045/187037591-1dd7399a-46ba-4320-a90a-74dbf3d939f3.png)

#### Type of change

### How to test it

1. Go to Login and access
2. Click on `Providers`.
3. Click on `Casa Rossello` (it is the only one that has products).
4. You can see the products

### Checklist:
